### PR TITLE
Add net6.0

### PR DESCRIPTION
**Pull request by oleg-n-k**
Add net6.0

### CODE REVIEW

File: HappyTravel.ConsulKeyValueClient.Abstractions/HappyTravel.ConsulKeyValueClient.Abstractions.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp3.1;net5.0</TargetFrameworks>
+    <TargetFrameworks>net5.0;net6.0;netcoreapp3.1</TargetFrameworks>
```
Comment (DFSko) [inline HappyTravel.ConsulKeyValueClient.Abstractions/HappyTravel.ConsulKeyValueClient.Abstractions.csproj:4]: Maybe drop 3.1?

File: HappyTravel.ConsulKeyValueClient.ConfigurationProvider/HappyTravel.ConsulKeyValueClient.ConfigurationProvider.csproj [hunk 2]
```diff
@@ -1,14 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <PackageVersion>1.6.0</PackageVersion>
-        <TargetFrameworks>net5.0;netcoreapp3.1</TargetFrameworks>
+        <PackageVersion>1.6.1</PackageVersion>
+        <TargetFrameworks>net5.0;net6.0;netcoreapp3.1</TargetFrameworks>
```
Comment (DFSko) [inline HappyTravel.ConsulKeyValueClient.ConfigurationProvider/HappyTravel.ConsulKeyValueClient.ConfigurationProvider.csproj:5]: Maybe drop 3.1?

File: HappyTravel.ConsulKeyValueClient.ConsulClient/HappyTravel.ConsulKeyValueClient.ConsulClient.csproj [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp3.1;net5.0</TargetFrameworks>
+    <TargetFrameworks>net5.0;net6.0;netcoreapp3.1</TargetFrameworks>
```
Comment (DFSko) [inline HappyTravel.ConsulKeyValueClient.ConsulClient/HappyTravel.ConsulKeyValueClient.ConsulClient.csproj:4]: Maybe drop 3.1?

File: HappyTravel.ConsulKeyValueClient.FileSettingsProvider/HappyTravel.ConsulKeyValueClient.FileSettingsProvider.csproj [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp3.1;net5.0</TargetFrameworks>
+    <TargetFrameworks>net5.0;net6.0;netcoreapp3.1</TargetFrameworks>
```
Comment (DFSko) [inline HappyTravel.ConsulKeyValueClient.FileSettingsProvider/HappyTravel.ConsulKeyValueClient.FileSettingsProvider.csproj:4]: Maybe drop 3.1?

File: HappyTravel.ConsulKeyValueClient.KvUpdater/HappyTravel.ConsulKeyValueClient.KvUpdater.csproj [hunk 5]
```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>netcoreapp3.1;net5.0</TargetFrameworks>
+    <TargetFrameworks>net5.0;net6.0;netcoreapp3.1</TargetFrameworks>
```
Comment (DFSko) [inline HappyTravel.ConsulKeyValueClient.KvUpdater/HappyTravel.ConsulKeyValueClient.KvUpdater.csproj:5]: Maybe drop 3.1?

File: HappyTravel.ConsulKeyValueClient/HappyTravel.ConsulKeyValueClient.csproj [hunk 6]
```diff
@@ -1,19 +1,19 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp3.1;net5.0</TargetFrameworks>
+    <TargetFrameworks>net5.0;net6.0;netcoreapp3.1</TargetFrameworks>
```
Comment (DFSko) [inline HappyTravel.ConsulKeyValueClient/HappyTravel.ConsulKeyValueClient.csproj:4]: Maybe drop 3.1?

**Review by DFSko — APPROVED** [review]

**Review by kirillta — CHANGES_REQUESTED** [review]
Please, exclude ` .DS_Store` files from the PR

**Review by hasmikharutyunyan — APPROVED** [review]

File: HappyTravel.ConsulKeyValueClient.Abstractions/HappyTravel.ConsulKeyValueClient.Abstractions.csproj (hunk 1 above)
Comment (oleg-n-k) [reply HappyTravel.ConsulKeyValueClient.Abstractions/HappyTravel.ConsulKeyValueClient.Abstractions.csproj]: fixed

File: HappyTravel.ConsulKeyValueClient.ConfigurationProvider/HappyTravel.ConsulKeyValueClient.ConfigurationProvider.csproj (hunk 2 above)
Comment (oleg-n-k) [reply HappyTravel.ConsulKeyValueClient.ConfigurationProvider/HappyTravel.ConsulKeyValueClient.ConfigurationProvider.csproj]: fixed

File: HappyTravel.ConsulKeyValueClient.ConsulClient/HappyTravel.ConsulKeyValueClient.ConsulClient.csproj (hunk 3 above)
Comment (oleg-n-k) [reply HappyTravel.ConsulKeyValueClient.ConsulClient/HappyTravel.ConsulKeyValueClient.ConsulClient.csproj]: fixed

File: HappyTravel.ConsulKeyValueClient.KvUpdater/HappyTravel.ConsulKeyValueClient.KvUpdater.csproj (hunk 5 above)
Comment (oleg-n-k) [reply HappyTravel.ConsulKeyValueClient.KvUpdater/HappyTravel.ConsulKeyValueClient.KvUpdater.csproj]: fixed

File: HappyTravel.ConsulKeyValueClient/HappyTravel.ConsulKeyValueClient.csproj (hunk 6 above)
Comment (oleg-n-k) [reply HappyTravel.ConsulKeyValueClient/HappyTravel.ConsulKeyValueClient.csproj]: fixed

**Review by kirillta — APPROVED** [review]

**Pull request merged**